app/controllers/categories_controller.rb

```diff
@@ -175,4 +183,8 @@ def category_params
     def fetch_category
       @category = Category.find_by(slug: params[:id]) || Category.find_by(id: params[:id].to_i)
     end
+
+    def initialize_staff_action_logger
```

Comment:
Hehe I was about to ask on meta whether we need to log categories that are deleted